pyright/alt-1/requirements.txt

```diff
@@ -0,0 +1,104 @@
+#
+# NOTE: It is important that dagster-* dependencies of any package listed here are also listed.
+# e.g. if examples/foo depends on python_modules/libraries/dagster-foo, then dagster-foo must be
+# listed if examples/foo is listed.
+#
+# NOTE: Paths are relative to repo root.
+
+### examples/assets_smoke_test
+# -e examples/assets_smoke_test
+#   -e python_modules/dagster[pyright,test]
+#   -e python_modules/dagit
+#   -e python_modules/dagster-graphql
+#   -e python_modules/libraries/dagster-pandas/
+#   -e python_modules/libraries/dagster-dbt/
+#   -e python_modules/libraries/dagster-snowflake/
+#   -e python_modules/libraries/dagster-snowflake-pandas/
+
+### examples/assets_pandas_type_metadata
+-e examples/assets_pandas_type_metadata
+  -e python_modules/dagster[pyright,test]
+  -e python_modules/dagit
+  -e python_modules/dagster-graphql
+  -e python_modules/libraries/dagster-pandera/
+
+### examples/project_fully_featured (and experimental version)
+-e examples/project_fully_featured
+-e examples/experimental/project_fully_featured_v2_resources
+  -e python_modules/dagster[pyright,test]
+  -e python_modules/dagit
+  -e python_modules/dagster-graphql
+  -e python_modules/libraries/dagster-aws/
+  -e python_modules/libraries/dagster-dbt/
+  -e python_modules/libraries/dagster-duckdb-pandas/
+    -e python_modules/libraries/dagster-duckdb/
+  -e python_modules/libraries/dagster-postgres/
+  -e python_modules/libraries/dagster-pyspark/
+  -e python_modules/libraries/dagster-slack/
+  -e python_modules/libraries/dagster-spark/
+
+### examples/quickstart_aws
+# -e examples/quickstart_aws
+#   -e python_modules/dagster[pyright,test]
+#   -e python_modules/dagit
+#   -e python_modules/dagster-graphql
+#   -e python_modules/libraries/dagster-aws/
+#   -e ../internal/dagster-cloud/python_modules/dagster-cloud
+#     -e python_modules/dagster[pyright,test]
+#     -e python_modules/dagit
+#     -e python_modules/dagster-graphql
+#     -e python_modules/libraries/dagster-aws/
+#     -e python_modules/libraries/dagster-docker/
+#     -e python_modules/libraries/dagster-k8s/
+#     -e python_modules/libraries/dagster-postgres/
+
+### examples/quickstart_etl
+# -e examples/quickstart_etl
+#   -e python_modules/dagster[pyright,test]
+#   -e python_modules/dagit
+#   -e python_modules/dagster-graphql
+#   -e ../internal/dagster-cloud/python_modules/dagster-cloud
+#     -e python_modules/dagster[pyright,test]
+#     -e python_modules/dagit
+#     -e python_modules/dagster-graphql
+#     -e python_modules/libraries/dagster-aws/
+#     -e python_modules/libraries/dagster-docker/
+#     -e python_modules/libraries/dagster-k8s/
+#     -e python_modules/libraries/dagster-postgres/
+
+### examples/quickstart_gcp
+# -e examples/quickstart_gcp
+#   -e python_modules/dagster[pyright,test]
+#   -e python_modules/dagit
+#   -e python_modules/dagster-graphql
+#   -e python_modules/libraries/dagster-gcp
+#   -e ../internal/dagster-cloud/python_modules/dagster-cloud
```

Comment:
no references to internal in the OSS repo pleaase

Reply:
Yeah this was there as a placeholder I intend to remove before merge (they're gone now)-- I was looking for a solution for `quickstart-*` examples, since they depend on `dagster-cloud`.

But having looked at the code of these, they don't actually import from `dagster_cloud` (or most of their deps), so it looks like they can successfully be type-checked in the master env without needing to install their deps.